server/public/style.css

```diff
@@ -3,6 +3,12 @@
   max-width: 75%;
 }
 
+.title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
Check out Bootstrap's flex helpers

https://getbootstrap.com/docs/4.4/layout/utilities-for-layout/